packages/destination-actions/src/destinations/salesforce/sf-operations.ts

```diff
@@ -1,8 +1,13 @@
 import { IntegrationError, RequestClient } from '@segment/actions-core'
 import type { GenericPayload } from './sf-types'
-import { mapObjectToShape } from './sf-object-to-shape'
+import { mapObjectToShape, snakeCaseToCamelCase } from './sf-object-to-shape'
 
 export const API_VERSION = 'v53.0'
+const isSettingsKey = new Map<string, boolean>([
+  ['operation', true],
+  ['traits', true],
+  ['externalIdFieldName', true]
+])
```

Comment:
Nitpick, can use a `Set` instead.

```suggestion
const isSettingsKey = new Set<string>(['operation', 'traits', 'externalIdFieldName'])
```